pkg/scheduler/scheduler.go

```diff
@@ -549,13 +545,14 @@ func (sched *Scheduler) scheduleOne(ctx context.Context) {
 		// preempt, with the expectation that the next time the pod is tried for scheduling it
 		// will fit due to the preemption. It is also possible that a different pod will schedule
 		// into the resources that were preempted, but this is harmless.
+		nominatedNode := ""
 		if fitError, ok := err.(*core.FitError); ok {
 			if sched.DisablePreemption {
 				klog.V(3).Infof("Pod priority feature is not enabled or preemption is disabled by scheduler configuration." +
 					" No preemption is performed.")
 			} else {
 				preemptionStartTime := time.Now()
-				sched.preempt(schedulingCycleCtx, prof, state, pod, fitError)
+				nominatedNode, _ = sched.preempt(schedulingCycleCtx, prof, state, pod, fitError)
```

Comment:
@ahg-g can you hold this PR until my preemption work is complete? This preempt() method may no longer needed in new codebase.

Reply:
But the concept is the same, the idea is to set the nominated name and the condition in the same patch request. Lets do this in the existing code and see how things will look like when this refactored, what do you think?

Reply:
SG. Let's finish this first.